app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :users, class_name: 'User'
+
+  def update_posts_counter
+    User.update(posts_counter: User.first.post.count)
+  end
+
+  def most_recent_comments
+    Comment.limit(5).order(created_at: :desc).where(post: self)
+  end
```

Comment:
- [ ] Post has comments, therefore, you don't have to explicitly add post: self, and it is better to add limit at the end, not sure if it works at the beginning:
```ruby
def recent_comments
  comments.order('created_at Desc').limit(5)
end
```